src/main/java/com/paskar/email/application/configuration/HibernateConfig.java

```diff
@@ -0,0 +1,53 @@
+package com.paskar.email.application.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+
+@Configuration
+@EnableTransactionManagement
+public class HibernateConfig {
+
+
+    @Bean
+    public PlatformTransactionManager transactionManager() {
+        JpaTransactionManager tm =
+                new JpaTransactionManager();
+        tm.setEntityManagerFactory(entityManagerFactory());
+        return tm;
+    }
+
+    @Bean
+    public EntityManagerFactory entityManagerFactory() {
```

Comment:
By default Spring Boot generates these classes instead of you.
You do not have to do it!